spec.html

```diff
@@ -40040,7 +40040,7 @@ <h1>TypedArray Objects</h1>
             <dfn>%BigInt64Array%</dfn>
           </td>
           <td>
-            ~BigInt64~
+            ~bigint64~
```

Comment:
For `~BigInt~` and `~BigInt64~` I opted for `~bigint~` and `~bigint64~` instead of, say, `~big-int~`.